flixel/system/debug/completion/CompletionList.hx

```diff
@@ -37,8 +36,6 @@ class CompletionList extends Sprite
 		createPopupEntries(capacity);
 		createScrollBar();
 		updateSelectedItem();
-
-		FlxG.stage.addEventListener(KeyboardEvent.KEY_DOWN, onKeyDown);
```

Comment:
Do you know why this wasn't working? Is this a `preventDefault` issue? or is the console blocking these events somehow?

Reply:
I'm unsure, I believe it's not console or `FlxG.stage` specific, since `Lib.current.addEventListener` also doesn't register tab key, and also the autofocus event listener doesn't work either ("Press TAB to focus the console!")